services/121-service/src/user/enum/permission.enum.ts

```diff
@@ -70,6 +70,9 @@ export enum PermissionEnum {
   // Registration(s) - Import
   RegistrationImportTemplateREAD = 'registration:import-template.read',
 
+  // Registration(s) - Duplication
+  RegistrationDuplicationDELETE = 'registration:duplication.delete',
```

Comment:
Where is this permission added to the "update duplication status"-endpoint?

Reply:
There should also be a migration to add this permission to the permission table and than also add it to a role (CvaManager) or else nobody is able to use it. Migration should be something like:

```
  await queryRunner.query(
    `INSERT INTO "121-service"."permission" ("name") VALUES ('registration:duplication.delete')`,
  );
 const permissionId = await queryRunner.query(
    `SELECT id FROM "121-service"."permission" where name = 'registration:duplication.delete',
  );
 const cvaRoleId =   await queryRunner.query(
    `SELECT id FROM "121-service"."user_role" where role = ${DefaultUserRole.CvaManager}`,
  );
 // Insert these 2 id's in the `user_role_permissions_permission` table, if the entry does not exist already
```

(Untested code)